iceoryx_posh/test/moduletests/test_popo_waitset.cpp

```diff
@@ -394,8 +420,35 @@ TEST_F(WaitSet_test, TimedWaitReturnsAllTriggeredConditionWhenAllAreTriggered)
     WaitReturnsAllTriggeredConditionWhenAllAreTriggered(this, [&] { return m_sut.timedWait(10_ms); });
 }
 
-void WaitReturnsTriggersWithCorrectCallbacks(WaitSet_test* test,
-                                             const std::function<WaitSet<>::EventInfoVector()>& waitCall)
+void WaitReturnsTriggersWithOneCorrectCallback(WaitSet_test* test,
```

Comment:
this violates our coding style. Functions must start lower case

Reply:
I was just checking if you pay attention ;)